src/Tribe/Repository.php

```diff
@@ -806,7 +806,7 @@ public function nth( $n ) {
 	 *
 	 * @return mixed A scalar value or a callable.
 	 */
-	public function apply_modifier( $key, $value ) {
+	public function apply_modifier( $key, $value = null ) {
```

Comment:
👏 I was wondering about this in regards to all the boolean where modifiers we built, the default `$bool = true` in the methods would never be able to get their default.